### PR TITLE
Pundit

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**

### CODE REVIEW

**Comment by josephbridgwaterrowe** [comment]
Please review, branch should be ready to merge... that being said I didn't give it any rubocop love.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
+[![Build Status](https://travis-ci.org/westernmilling/chancellor.svg)](https://travis-ci.org/westernmilling/chancellor)
```
Comment (michaelachrisco) [inline README.md]: Might use this myself :+1:

File: db/migrate/20150107230109_add_is_active_to_user.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class AddIsActiveToUser < ActiveRecord::Migration
+  def change
+    # Integer for boolean values since booleans are just stupid
```
Comment (michaelachrisco) [inline db/migrate/20150107230109_add_is_active_to_user.rb]: For pushes to master, I think comments like this should stay within the commit. ActiveRecord is pretty dumb code. Push request in order?

**Comment by michaelachrisco** [comment]
Looks good other than a few things that hound picked up on and the comment.

**Pull request merged**